doc/admin-guide/plugins/tcpinfo.en.rst

```diff
@@ -103,6 +103,31 @@ The following options may be specified in :file:`plugin.config`:
   not required to be in the configuration file.  To achieve a log rate
   of 1% you would set this value to 10.
 
+--rolling-enabled=VALUE
+  This logfile option allows you to set logfile rolling behaviour of
+  the ``tcp-metrics.log`` without making any changes to the global
+  logging configurations.  This option overrides the 
+  ``proxy.config.output.logfile.rolling_enabled`` setting in records.config
```

Comment:
Use the `ts:cv` notation here to link to the correct configuration variable documentation.